File: 217-contains-duplicate.cpp
#include <algorithm>
#include <iterator>
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        for(int i = 0; i < nums.size(); i++){
            auto iter = std::next(nums.begin(),i+1);
            if(std::find(iter, nums.end(), nums[i])!=nums.end())
                return true;
        }
        return false;
    }
};

File: 217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        return True if len(nums) != len(set(nums)) else False


class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        unique_dict = {}
        for num in nums:
            if num in unique_dict.keys():
                return True
            else:
                unique_dict[num] = 1
        return False
